setup.py

```diff
@@ -225,6 +226,40 @@ def run(self):
             sdist.run(self)
 
 
+def get_build_overrides():
+    """
+    Custom build commands to add `--std=c99` to compilation
+    """
+    from numpy.distutils.command.build_clib import build_clib
+    from numpy.distutils.command.build_ext import build_ext
+
+    def _is_using_gcc(obj):
+        is_gcc = False
+        if obj.compiler.compiler_type == 'unix':
+            cc = sysconfig.get_config_var("CC")
+            if not cc:
+                cc = ""
+            compiler_name = os.path.basename(cc)
+            is_gcc = "gcc" in compiler_name
+        return is_gcc
+
+    class new_build_clib(build_clib):
+        def build_a_library(self, build_info, lib_name, libraries):
+            if _is_using_gcc(self):
+                args = build_info.get('extra_compiler_args') or []
+                args.append('--std=c99')
+                build_info['extra_compiler_args'] = args
+            build_clib.build_a_library(self, build_info, lib_name, libraries)
+
+    class new_build_ext(build_ext):
+        def build_extension(self, ext):
+            if _is_using_gcc(self):
+                if '--std=c99' not in ext.extra_compile_args:
+                    ext.extra_compile_args.append('--std=c99')
```

Comment:
Isn't the option `-std=c99` with single dash?

Reply:
fixing